https://github.com/ansible/ansible/pull/56378

ansible/modules/packaging/os/pkg5.py, line 183, in <module>
ansible/modules/packaging/os/pkg5.py, line 106, in main
ansible/modules/packaging/os/pkg5.py, line 158, in ensure

TypeError: can only concatenate list (not "filter") to list

--- ansible-2.9.11/lib/ansible/modules/packaging/os/pkg5.py	2020-07-21 04:08:11.000000000 +0000
+++ ansible-2.9.11/lib/ansible/modules/packaging/os/pkg5.py	2020-08-06 13:22:07.330885464 +0000
@@ -152,7 +152,7 @@
     else:
         no_refresh = ['--no-refresh']
 
-    to_modify = filter(behaviour[state]['filter'], packages)
+    to_modify = list(filter(behaviour[state]['filter'], packages))
     if to_modify:
         rc, out, err = module.run_command(['pkg', behaviour[state]['subcommand']] + dry_run + accept_licenses + beadm + no_refresh + ['-q', '--'] + to_modify)
         response['rc'] = rc
